services/wallet/transfer/transaction_manager_multitransaction.go

```diff
@@ -39,14 +39,21 @@ func (tm *TransactionManager) CreateMultiTransactionFromCommand(command *MultiTr
 
 	multiTransaction := multiTransactionFromCommand(command)
 
-	// Set network for single chain transactions
+	// Extract network from args
 	switch multiTransaction.Type {
 	case MultiTransactionSend, MultiTransactionApprove, MultiTransactionSwap:
 		if multiTransaction.FromNetworkID == wallet_common.UnknownChainID && len(data) == 1 {
 			multiTransaction.FromNetworkID = data[0].ChainID
 		}
 	case MultiTransactionBridge:
-		break
+		if len(data) == 1 {
```

Comment:
@Cuteivist just to be safe, could you add
```suggestion
		if len(data) == 1 && data[0].HopTx != nil {
```

Reply:
Though I don't like this approach at all :) and hope to change it soon.

Reply:
Done.
It is fix to current implementation that we will be changing, so I didn't go deep into refactoring both status-desktop and status-go to get it in better shape.